### DISCUSSION TITLE
Any plans to have sharding?

### DISCUSSION THREAD

**Discussion by NikolayS** [hydradatabase/hydra#41] [General]
If I have 100+ TiB of data, will I be able to use Hydra efficiently? How?

**Comment by wuputah** [comment]
It's in our long-term plans, generally requires separation of compute and storage. Today we are limited by maximum volume sizes, though we can go up to 64 TiB _uncompressed_ on AWS, which can potentially store 10x or more when compressed. You can also export and store your data to S3 and use Hydra to access that data (use Hydra as a lakehouse).
